controllers/storagecluster/cephtoolbox.go

```diff
@@ -31,11 +31,21 @@ func (r *StorageClusterReconciler) ensureToolsDeployment(sc *ocsv1.StorageCluste
 		Effect:   corev1.TaintEffectNoSchedule,
 	}}
 
-	tolerations = append(tolerations, sc.Spec.ManagedResources.CephToolbox.Tolerations...)
+	// Get the ocsinitialization CR
+	ocsinit := &ocsv1.OCSInitialization{}
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: "ocsinit", Namespace: namespace}, ocsinit)
+	if err != nil && !errors.IsNotFound(err) {
+		return err
+	}
+
+	// Get tolerations from both the storagecluster CR & ocsinitialization CR
+	// Keeping this for ability to add tolerations from ocsinitialization CR
+	tolerations = append(tolerations, sc.Spec.ManagedResources.CephToolbox.Placement.Tolerations...)
+	tolerations = append(tolerations, ocsinit.Spec.Tolerations...)
```

Comment:
It'd be better to merge it such that there are no duplicate toleration.

Reply:
Done